packages/zoe/src/contracts/otcDesk.js

```diff
@@ -0,0 +1,117 @@
+// @ts-check
+
+import { E } from '@agoric/eventual-send';
+import { assert, details } from '@agoric/assert';
+import { trade } from '../contractSupport';
+
+import '../../exported';
+
+/**
+ * The creator gets a creatorFacet that allows them to add inventory, make quotes and
+ * create specific invitations to exercise those quotes.
```

Comment:
Please add more details on the usage of this contract. What shape proposals are accepted? What methods are in the creatorFacet? What do the invitations that the creator can hand out allow? What can recipients of those invitations rely on? 
`makeQuote` seems to have some interesting functionality. How is it used? 
Can the recipient of a `removeInventory` inventory verify that the inventory is reserved for their use?

Reply:
Done